Cargo.toml

```diff
@@ -12,7 +12,7 @@ async-stream           = { version = "0.3" }
 axum                   = { version = "0.7" }
 configured             = { version = "0.7" }
 error-ext              = { version = "0.2" }
-evented                = { version = "0.2" }
+evented                = { path = "." }
```

Comment:
I guess you had that to keep the example more standalone? Anyway depending on a released version will make updating any dependencies challenging. I think either examples should not be a workspace member or should depend on current version of evented but not both ;)